app/services/get_users_service.rb

```diff
@@ -5,15 +5,22 @@ class GetUsersService < ActiveInteraction::Base
   string :filter_criterion, default: 'user_name'
 
   def execute
+    if User.count.eql? 0
```

Comment:
why we need this check?

Reply:
There is no sense to implement this service's algorithm if users count is 0. So, I decided to separate this case.

Reply:
Can it be `return if User.count.eql? 0`

Reply:
Yes